### ISSUE TITLE
Call to undefined function set_exit_overload()

### ISSUE THREAD

**Issue by nytche** [php-test-helpers/php-test-helpers#26]
hello,

I installed test_helpers, validated with phpinfo(), and tested with your example code from Github, but i have the error pasted in the title. Did i missed something?

running in PHP 5.5.8 with Cli.

**Comment by jimlei** [comment]
In case someone else get this.

I enabled the extension here as I needed to do the work around because of xdebug.
/etc/php5/apache2/conf.d/99-test-helpers.ini

This gave me the same behaviour as nicocans, phpinfo says everything is ok while phpunit/cli will not work.

Simply copying the file from
/etc/php5/apache2/conf.d/99-test-helpers.ini
to
/etc/php5/cli/conf.d/99-test-helpers.ini

should do the trick.
